src/geo/transform.js

```diff
@@ -23,8 +23,8 @@ class Transform {
         this.tileSize = 512; // constant
 
         this._renderWorldCopies = renderWorldCopies === undefined ? true : renderWorldCopies;
-        this._minZoom = minZoom || 0;
-        this._maxZoom = maxZoom || 22;
+        this.minZoom = minZoom || 0;
+        this.maxZoom = maxZoom || 22;
```

Comment:
Good question. When you use the following declaration `this._minZoom = 1` the value is declared directly but when you use `this.minZoom = 1` the setter of minZoom is called. If I wouldn't have changed this the error wouldn't be thrown when `maxZoom > minZoom` when initializing the map, only when changing the minZoom and maxZoom after the initialization.